zerver/tests/test_new_users.py

```diff
@@ -113,30 +113,30 @@ class TestBrowserAndOsUserAgentStrings(ZulipTestCase):
     def setUp(self) -> None:
         super().setUp()
         self.user_agents = [
-            ('mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) ' +
+            ('Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) ' +
                 'Chrome/54.0.2840.59 Safari/537.36', 'Chrome', 'Linux',),
-            ('mozilla/5.0 (windows nt 6.1; win64; x64) applewebkit/537.36 (khtml, like gecko) ' +
-                'chrome/56.0.2924.87 safari/537.36', 'Chrome', 'Windows',),
-            ('mozilla/5.0 (windows nt 6.1; wow64; rv:51.0) ' +
-                'gecko/20100101 firefox/51.0', 'Firefox', 'Windows',),
-            ('mozilla/5.0 (windows nt 6.1; wow64; trident/7.0; rv:11.0) ' +
-                'like gecko', 'Internet Explorer', 'Windows'),
+            ('Mozilla/5.0 (Windows NT 6.1; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) ' +
+                'Chrome/56.0.2924.87 Safari/537.36', 'Chrome', 'Windows',),
+            ('Mozilla/5.0 (Windows NT 6.1; WOW64; rv:51.0) ' +
+                'Gecko/20100101 Firefox/51.0', 'Firefox', 'Windows',),
+            ('Mozilla/5.0 (Windows NT 6.1; WOW64; Trident/7.0; rv:11.0) ' +
+                'like Gecko', 'Internet Explorer', 'Windows'),
```

Comment:
Corrected the casing to match with the actual user agent, as `user-agents` module returns `other` when they are in lower case.